sdk/python/kfp/dsl/_pipeline_param.py

```diff
@@ -21,6 +21,16 @@
 # For now, this identifies a condition with only "==" operator supported.
 ConditionOperator = namedtuple('ConditionOperator', 'operator operand1 operand2')
 
+def _extract_pipeleineparam(payload: str):
```

Comment:
Hello! Maybe fix the name? `_extract_pipeleineparam` to `_extract_pipelineparam`

Reply:
Thanks for catching this.

Reply:
Fixed the typo.